doc-Self_Service_User_Interface_Guide/topics/My_Services_Tab.adoc

```diff
@@ -4,17 +4,19 @@
+On the same screen, you have the option to edit the service name and description, edit tags, set ownership, retire the service now or at a later date, or remove the service by clicking more options _(represented by three dots)_ next to the service. You also have options to perform power operations, that is start, stop, or suspend, at the service level. 
 
 You can perform most of the above tasks in bulk (except editing service details and power operations). To do so, select multiple services and then click the designated button for the required task on the upper right corner of the screen.    
 
 image:sui-services.png[]
 
-Click on a service to see more details, including information about associated virtual machines and snapshots under *Resources*, and associated parent catalog item under *Relationships*. You can click more options _(represented by three dots)_ to perform power operations at the resource level, or you can click *View Details* (or on the virtual machine name) to see more information about the resource, including compliance, retirement, timeline of power events, utilization, Smart State analysis, disk usage, and perform power operations. 
+Click on a service to see more details, including information about associated virtual machines and snapshots under *Resources*, and associated parent catalog item under *Relationships*.
 
-Click *Snapshots* to view available snapshots, or create or delete snapshots via the *Configuration* button on the next screen. A snapshot is a copy of the virtual machine's disk file (VMDK) that captures the entire state of the virtual machine at a point in time. 
+.Virtual Machine dashboard:
 
-// Use *Revert* under more options _(represented by three dots)_ to return a virtual machine to a previous state when you took the snapshot.
+You can click on the virtual machine name or *View Details* under more options _(represented by three dots)_, to see the virtual machine dashboard. The dashboard shows more information of the virtual machine resource, including compliance, retirement, snapshots, timeline of power events, utlilization of CPU, Memory, and Storage, Smart State analysis, tags, custom attributes, disk usage. Additionally, you can also perform power operations at the resource level, and view and create snapshots. 
+
+Click *Snapshots* to view available snapshots, or create or delete snapshots via the *Configuration* button on the next screen. A snapshot is a copy of the virtual machine's disk file (VMDK) that captures the entire state of the virtual machine at a point in time. Click *Revert* under more options _(represented by three dots)_ to return a virtual machine to a previous state when you took the snapshot.
```

Comment:
If I click on My Services ->VM Name and I'm looking at the page with Properties/Resources/Relationships, I see both a *Snapshot* and *Configuration* button, so its a little unclear to me what the next screen would be where I'd find another configuration button? In this instance, have I first clicked *Snapshot* to view an existing snapshot? 

I might specify that the _revert_ option is only located on the *snapshot* summary page. etc

Reply:
Hey Chris, thanks for your comment. I have removed that line (meant to do that earlier - because dedicated procedures explaining how to go about it now exists further below in that section) as well as restructured and made a few more updates along the way, including screen captures. Let me know what you think. Thanks!